native_client/deepspeech.cc

```diff
@@ -273,7 +311,13 @@ StreamingState::finishStream()
     processBatch(batch_buffer, batch_buffer.size()/model->mfcc_feats_per_timestep);
   }
 
-  return model->decode(accumulated_logits);
+  if (extended_output) {
+    vector<Output> out = model->decode_raw(accumulated_logits);
+    std::vector<META_DICT> word_list = model->metadata_from_output(out[0],accumulated_logits.size());    
+    return model->json_output_from_metadata(word_list);
```

Comment:
I'm really not okay with that: you are changing the meaning of the output here, exposing a JSON if we pass some specific parameters. That really feels weird from an external point of view.